app/lib/kafka.server.ts

```diff
@@ -13,11 +13,16 @@ import { domain, getEnvOrDieInProduction } from './env.server'
 
 const client_id = getEnvOrDieInProduction('KAFKA_CLIENT_ID') ?? ''
 const client_secret = getEnvOrDieInProduction('KAFKA_CLIENT_SECRET')
-const kafka = new Kafka({
-  client_id,
-  client_secret,
-  domain,
-})
+let kafka: Kafka | undefined = undefined
+try {
+  kafka = new Kafka({
+    client_id,
+    client_secret,
+    domain,
+  })
+} catch (e) {
+  console.warn('Failed to initialize Kafka: ', e)
```

Comment:
I don't think that creating a Kafka object actually does anything over the network, does it?

Reply:
Something is up with the invoked lambda attempting to do something with kafka. I am trying to find out exactly where this is happening